src/components/renderer.cpp

```diff
@@ -310,6 +300,24 @@ void renderer::end() {
     fill_background();
   }
 
+
+  if (m_pseudo_transparency) {
+    cairo_pattern_t* barcontents{};
+    m_surface->write_png("/tmp/test.png");
```

Comment:
Is this debugging code or is this actually used?

Reply:
Oops yes obviously the write_PNG is debugging, good catch